src/app/main-menu/main-menu.component.html

```diff
@@ -0,0 +1,12 @@
+<mat-toolbar>
+  <div [ngClass]="'logo'">
+    <img [src]="pathSVG" [alt]="altLogo">
+  </div>
+  <span class="example-spacer"></span>
```

Comment:
Klasy powinny nieść ze sobą jakieś znaczenie - być semantyczne. 
```suggestion
  <span class="menu-spacer"></span>
```